tournaments/index.js

```diff
@@ -794,6 +794,7 @@ var commands = {
 			if (params.length < 1) {
 				return this.sendReply("Usage: " + cmd + " <minutes|off>");
 			}
+			if (params[0] === 'Infinity' || params[0] === '0') params[0] = 'off';
```

Comment:
Because they're parsed as NaN. ``Infinity``'s the only case where it parses as Infinity